admin-core/src/main/java/com/depromeet/domain/franchise/infrastructure/persistence/FranchiseRepositoryImpl.java

```diff
@@ -26,19 +27,27 @@ public List<Franchise> findAll() {
 
     @Override
     public void save(final Franchise franchise) {
-        final FranchiseEntity franchiseEntity = Franchise.toEntity(franchise);
+        final var franchiseEntity = Franchise.toEntity(franchise);
         franchiseEntityRepository.save(franchiseEntity);
     }
 
     @Override
     public Optional<Franchise> findByName(final String name) {
-        final Optional<FranchiseEntity> findFranchiseEntity = franchiseEntityRepository.findByName(name);
+        final var findFranchiseEntity = franchiseEntityRepository.findByName(name);
         return findFranchiseEntity.map(FranchiseEntity::toDomain);
     }
 
     @Override
-    public Optional<Franchise> findById(long id) {
-        final Optional<FranchiseEntity> findFranchiseEntity = franchiseEntityRepository.findById(id);
-        return findFranchiseEntity.map(FranchiseEntity::toDomain);
+    public Franchise findById(final long id) {
```

Comment:
Long 대신 long으로 하신 이유가 있으실까요??

```suggestion
    public Franchise findById(final Long id) {
```

Reply:
수정했습니다~~ 감사합니다!